### PR TITLE
README: Add menu

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @qiuyesuifeng @iamxy PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
-# docs-cn
+# TiDB 文档 
+
```
Comment (qiuyesuifeng) [inline README.md]: Should we translate `TiDB/TiKV/PD documents in Chinese.` into Chinese too?

Comment (shenli) [inline README.md]: I'm not sure about this. I keep this to make someone who does not speak Chinese could understand what's this repo used for.

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**